keras_contrib/applications/ror.py

```diff
@@ -0,0 +1,322 @@
+                                  'at ~/.keras/keras.json.')
+                    convert_all_kernels_in_model(model)
+            else:
+                if include_top:
+                    weights_path = get_file('ror_wrn_40_2_tf_kernels_tf_dim_ordering_no_top.h5',
+                                            TF_WEIGHTS_PATH,
+                                            cache_subdir='models')
+                else:
+                    weights_path = get_file('ror_wrn_40_2_tf_kernels_tf_dim_ordering_no_top.h5',
+                                            TF_WEIGHTS_PATH_NO_TOP,
+                                            cache_subdir='models')
+
+                model.load_weights(weights_path)
+
+                if K.backend() == 'theano':
+                    convert_all_kernels_in_model(model)
+
+    return model
+
+
+def __initial_conv(input):
+    x = Convolution2D(16, 3, 3, border_mode='same', init='he_normal')(input)
+    return x
+
+
+def __conv1_block(input, k=1, dropout=0.0, initial=False):
+    init = input
+
+    channel_axis = 1 if K.image_dim_ordering() == "th" else -1
+
+    # Check if input number of filters is same as 16 * k, else create convolution2d for this input
+    if initial:
+        if K.image_dim_ordering() == "th":
+            init = Convolution2D(16 * k, 1, 1, init='he_normal', border_mode='same')(init)
+        else:
+            init = Convolution2D(16 * k, 1, 1, init='he_normal', border_mode='same')(init)
+
+    x = BatchNormalization(axis=channel_axis)(input)
+    x = Activation('relu')(x)
+    x = Convolution2D(16 * k, 3, 3, border_mode='same', init='he_normal')(x)
+
+    if dropout > 0.0:
+        x = Dropout(dropout)(x)
+
+    x = BatchNormalization(axis=channel_axis)(x)
+    x = Activation('relu')(x)
+    x = Convolution2D(16 * k, 3, 3, border_mode='same', init='he_normal')(x)
+
+    m = merge([init, x], mode='sum')
+    return m
+
+
+def __conv2_block(input, k=1, dropout=0.0):
+    init = input
+
+    channel_axis = 1 if K.image_dim_ordering() == "th" else -1
+
+    # Check if input number of filters is same as 32 * k, else create convolution2d for this input
+    if K.image_dim_ordering() == "th":
+        if init._keras_shape[1] != 32 * k:
+            init = Convolution2D(32 * k, 1, 1, init='he_normal', border_mode='same')(init)
+    else:
+        if init._keras_shape[-1] != 32 * k:
+            init = Convolution2D(32 * k, 1, 1, init='he_normal', border_mode='same')(init)
+
+    x = BatchNormalization(axis=channel_axis)(input)
+    x = Activation('relu')(x)
+    x = Convolution2D(32 * k, 3, 3, border_mode='same', init='he_normal')(x)
+
+    if dropout > 0.0:
+        x = Dropout(dropout)(x)
+
+    x = BatchNormalization(axis=channel_axis)(x)
+    x = Activation('relu')(x)
+    x = Convolution2D(32 * k, 3, 3, border_mode='same', init='he_normal')(x)
+
+    m = merge([init, x], mode='sum')
+    return m
+
+
+def __conv3_block(input, k=1, dropout=0.0):
```

Comment:
function __conv2_block() and __conv3_block() are almost the same, only different on the number of feature maps, right? So please merge them into one function with the number of feature maps as a parameter.